ibodata/depth_profile.py

```diff
@@ -1,6 +1,37 @@
+import numpy as np
+
 from beprof.profile import Profile
 
 
 class DepthProfile(Profile):
```

Comment:
range method is missing

Reply:
it would be stupid to name method calculating range as `range` as this would introduce conflict with python range keyword

what about naming it `ran` ?